gunrock/graph/csr_parallel_iterator.cuh

```diff
@@ -34,7 +34,7 @@ struct ParallelIterator<VertexT, SizeT, ValueT, FLAG, HAS_CSR> {
       graph::Csr<VertexT, SizeT, ValueT, FLAG & graph::HAS_CSR_MASK,
                  cudaHostRegisterDefault, (FLAG & graph::HAS_CSR) != 0>;
 
-  __host__ __device__ ParallelIterator(VertexT v, CsrT* graph)
+  __host__ __device__ ParallelIterator(const VertexT v, const CsrT* graph)
```

Comment:
Let's consider naming this something like `ConstParallelIterator`

Reply:
How does this (https://github.com/gunrock/gunrock/pull/752/commits/40e9032b4dc4bcfca4b8ff887e887bfa7a4317fc) look?